README.md

```diff
@@ -36,6 +59,10 @@ published on the repo GitHub pages.
     shell.
 - **For m1 users**: Install [Protocol Buffers](https://grpc.io/docs/protoc-installation/)
```

Comment:
Is this really needed? I don't remember installing it and I have built successfully the engines.

Reply:
I'm also curious as to what this is actually for, I didn't need this (on m2). Also is this is m1 specific or ARM in general?